src/routes/apiRoutes.ts

```diff
@@ -7,6 +7,7 @@ router.get("/members/notpaid", apiController.getMembersNotPaid.bind(apiControlle
 router.get("/members/paymentStatus", apiController.getMembersPaymentStatus.bind(apiController));
 router.get("/members", apiController.getMembers.bind(apiController));
 router.get("/events", apiController.getEvents.bind(apiController));
+router.get("/events/specific", apiController.getSpecificEventWithMembers.bind(apiController));
```

Comment:
```suggestion
router.get("/events/:eventId", apiController.getSpecificEventWithMembers.bind(apiController));
```
Zukunftssicherer wenn die Id mitgegeben wird

in apiController.ts müsste dann noch so etwas wie
`
const event = await (clubStore.getSpecificGroupWithMembers(req.params.eventId));
`
stehen